lib/experimental/fonts/font-library/class-wp-font-collection.php

```diff
@@ -138,34 +138,22 @@ public static function is_config_valid( $config ) {
 			( empty( $config['src'] ) && empty( $config['font_families'] ) ) ||
 			( ! empty( $config['src'] ) && ! empty( $config['font_families'] ) )
 		) {
-			_doing_it_wrong( __METHOD__, __( 'Font Collection config "src" option OR "font_families" option are required.', 'gutenberg' ), '6.5.0' );
+			_doing_it_wrong(
+				__METHOD__,
+				sprintf(
+					/* translators: %1$s: src, %2$s: font_families */
+					__( 'Font Collection config "%1$s" option OR "%2$s" option are required.', 'gutenberg' ),
```

Comment:
```suggestion
					__( 'Font Collection config "%1$s" option OR "%2$s" option is required.', 'gutenberg' ),
```

Reply:
What happens if both `src` and `font_families` are defined?

Reply:
Also, let's make the [error above](https://github.com/WordPress/gutenberg/pull/58222/files#diff-17dde8701de56a0be9be544c9fcd4f0c1aec015f6e864808e8e52eb909b40927R130) consistent and also quote the config key:

```
'Font Collection config "%s" is required as a non-empty string.'
```

Reply:
Added the change of the string.

> What happens if both src and font_families are defined?

I think that's more unrelated to the Rest API and it's more about the PHP so we can discuss/  improve it in another PR.